privacyidea.py

```diff
@@ -81,7 +81,10 @@ def init(self, configurationAttributes):
 
         privacyidea_url = configurationAttributes.get("privacyidea_url").getValue2()
 
-        builder = PrivacyIDEA.Builder(privacyidea_url)
+        privacyidea_ua = configurationAttributes.get("privacyidea_user-agent").getValue2()
```

Comment:
why would we allow anything else than privacyIDEA-Gluu?

Reply:
You are totally right. But we do not tell anyone! ;-)
OK, I set  a constant, I think like we have done in all other occurances.

Reply:
Done in de7adba.
You may *squash* and merge.